src/main/java/duke/Task.java

```diff
@@ -0,0 +1,127 @@
+package duke;
+
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+
+/**
+ * Class task that duke.Duke creates
+ */
+public class Task {
+
+    String time = "";

```

Comment:
If you are talking about the fields in class `Task` such as `time`, it is because other classes in this package will read it. Would making it private and having a `getter()` be better?

Reply:
Hmm think in your case that might be a good idea? Since the `Task.java` is inside the package duke and not in a special package like duke.tasks